src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsTrackingOptInRequestsMobileWiki.java

```diff
@@ -164,10 +156,10 @@ public void adsTrackingRejectedForA9(String[] instantGlobals, List<String> urlPa
   )
   public void adsTrackingRejectedForComscoreAndQuantcast(List<String> urlPatterns) {
```

Comment:
What do you think of having two methods? `adsTrackingRejectedForComscore` and `adsTrackingRejectedForQuantcast`? It will help easily find out why the job is failing once we read the method's name which failed.